main.py

```diff
@@ -949,29 +982,44 @@ def handle_evc_deployed(self, event):
             return
         evc.try_setup_failover_path()
 
-    @listen_to("kytos/mef_eline.cleanup_evcs_old_path")
+    @listen_to("kytos/mef_eline.cleanup_evcs_old_path", pool="dynamic_single")
```

Comment:
Did you hit any other out of order event issues? 

`dynamic_single` is suitable for many cases, but notice here, this handler can be a bit IO heavy since it can handle many EVCs and it would do a request in a single thread, so the IO throughput would be severely impacted in cases where this handle here gets many events queued. Let's try to understand which cases you've seen that were problematic to see what else we can do here.

Reply:
This changed was because I had two source for `cleanup_evcs_old_path` to be triggered. I changed it back since `old_paths` is now a single path which flows are sent through an event.